jjb/jobs/rho-nightly-remote-scan.yaml

```diff
@@ -23,13 +23,16 @@
     builders:
         - config-file-provider:
             files:
-              - file-id: '{config-file-id}'
+              - file-id: '62cf0ccc-220e-4177-9eab-f39701bff8d7'
                 target: 'camayoc/config.yaml'
         - shining-panda:
             build-environment: virtualenv
             python-version: System-CPython-3
             nature: shell
+            system-site-packages: true
```

Comment:
@kdelee now it is including the system-site-packages

Reply:
Cool, I think that we should be good then